include/internal/refcount.h

```diff
@@ -176,11 +173,14 @@ typedef int CRYPTO_REF_COUNT;
 #  define REF_ASSERT_ISNT(i)
 # endif
 
-# ifdef REF_PRINT
-#  define REF_PRINT_COUNT(a, b) \
-        fprintf(stderr, "%p:%4d:%s\n", (void*)b, b->references, a)
+# ifdef REF_DEBUG
```

Comment:
Please drop `REF_DEBUG`, it's only redundant; the OSSL_TRACE macros are already conditionally defined.

You claimed, in https://github.com/openssl/openssl/pull/15353#discussion_r636993742, that `REF_DEBUG` (formerly `REF_PRINT`) is used for `REF_ASSERT_ISNT`, but if you look, just a few lines up, you'll find that isn't true.